gradle.properties

```diff
@@ -189,5 +189,29 @@ jfigletVersion=0.0.8
 
 embeddedRedisVersion=0.6
 
+#webjars versions
```

Comment:
Right now those variables are used in dependencies.gradle but the same version numbers are also hard coded in the URIs of most of the js/css artifacts in static-uris.properties.  Aren't all dependency versions set via a variable in gradle.properties? I was trying to follow that pattern. At one point I had also thought to filter those variables into the URIs contained in static-uris.properties so that one could change the version in one place. If I move static-uris.properties to another jar (instead of resources) then filtering would only need to be turned on for that one module's resource directory, but as a maven user it will probably take me some time to figure out how to do it in gradle so if you don't like filtering, I won't look into it.